README.md

```diff
@@ -23,3 +24,8 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+Answer: The $\Theta$ time complexity of my implementation would be $\Theta(V * E)$.
```

Comment:
Not sure how you're getting to this complexity. Looks like you're only iterating over nodes?

Reply:
I am iterating through all of the vertices in the unvisited array, and for each vertex I am iterating through the edges.

Reply:
All edges?

Reply:
Yes it does go through all edges which made me realize my algorithm is wrong. It finds the shortest path overall it does not select the shortest path at each node.

Reply:
But does it go through all edges for each node?

Reply:
Yes it iterates through all of the edges for each node.

Reply:
All of the edges attached to the node, but not *all* edges?

Reply:
Well it iterates through the edges attached to the vertex, and it does this for each node in the unvisited array until the unvisited arrray is empty, so wouldn't that iterate through all of the edges?

Reply:
Yes. But when you're multiplying, you're saying that you're iterating over all edges for each node.

Reply:
Yes I see what you are saying now. So it would be $(V + E)$ instead.